deploy/converged/cluster_role.yaml

```diff
@@ -821,3 +821,70 @@ rules:
   - get
   - patch
   - update
+
+---
```

Comment:
I see now the downside to including generated files in version control. All of the existing (Cluster)Role(Binding)s, CRDs, Deployments, etc are generated using the `hack/build-manifests.sh` and the `tools/manifest-templator/manifest-templator.go`.

In the end, the way the HCO is going to consume components (like the web-ui) is by depending on your CRD using OLM. Right now, if you want to properly add web-ui to the HCO you will need to create a new pkg (like `pkg/components` as an example, this would probably fit best in `pkg/uicomponents`) and make the appropriate changes to the `manifest-templator` as well as the associated templates under `templates/`.

Reply:
Honestly, this sounds like nightmare to maintain - to keep both yaml and there "golang rewrites" in sync.

Reply:
I definitely prefer the "In the end" part where we just consume the components. Maybe we should work towards that (ie. hold off on integrating directly with the HCO and focus on getting a CSV for web-ui that the HCO can consume)?